dataactcore/interfaces/function_bag.py

```diff
@@ -332,11 +335,18 @@ def check_job_dependencies(job_id):
                 # so it is eligible to be set to a 'ready'
                 # status and added to the queue
                 mark_job_status(dep_job_id, 'ready')
-                # add to the job queue
-                logger.info('Sending job %s to job manager', dep_job_id)
-                # will move this later
-                from dataactcore.utils.jobQueue import enqueue
-                enqueue.delay(dep_job_id)
+
+                # If local, utilize local db to mock queue
+                if g.is_local:
+                    new_queue_entry = SQS(job_id=dep_job_id)
+                    sess.add(new_queue_entry)
+                    sess.commit()
```

Comment:
One way to avoid these logic branches here is to create an interface that matches SQS's. Something like:
```python
class SQSMockQueue:
    @staticmethod
    def send_message(MessageBody):    # noqa
        sess = GlobalDB.db().session
        sess.add(SQS(job_id=MessageBody))
        sess.commit()

    @staticmethod
    def receive_messages(WaitTimeSeconds):  # noqa
        for sqs in sess.query(SQS):
            yield SQSMockMessage(sqs)

class SQSMockMessage:
    def __init__(self, sqs):
        self.sqs = sqs
        self.body = sqs.job_id

    def delete(self):
        sess = GlobalDB.db().session
        sess.delete(self.sqs)
        sess.commit()


def sqs_queue():
    if CONFIG_SOMETHING['local']:
        return SQSMockQueue
    else:
        # stuff that's in get_queue
        return queue
```